deviceModeInit.js

```diff
@@ -345,10 +376,15 @@ var rudderTracking = (function () {
     };
     if (pages[path] === "Registration Viewed") {
       rudderanalytics.track(pages[path], properties);
-      rs$("#create_customer").submit(userRegistered);
+      if (!disableClientIdentifierEvents) {
+        rs$("#create_customer").submit(userRegistered);
```

Comment:
[Q] Can we put this condition inside `submit` method itself ? Or may be this is jquery method, so we can't do anything here ?!

Reply:
Yes we can put `submit` inside  but can't understand the usage of going into one more specific level

Reply:
The intention was to abstract how we use `disableClientIdentifierEvents`. Everywhere there is a submit we now would need to put the if-condition right ?! Wanted to avoid it to be at a single place.

Reply:
But this is fine!

Reply:
Yes I thought of it as well but it might be possible that out of many request some get failed resulting in vague data